public/src/node.js

```diff
@@ -0,0 +1,146 @@
+// # Analyzing Data with Node
+//
+// As mentioned in the [introduction](getting_started.html), this guide is mostly geared for client-side data analysis, but with a few augmentations, the same tools can be readily used server-side with [Node](https://nodejs.org/).
+//
+// And the data is too large, this might in fact be your only option if you want to use JavaScript for your data analysis. None of your users will wait 5 minutes on your site while a giant file is loaded up, no matter how cool the analysis.
```

Comment:
yes - great suggestion. Incorrectly, I have in my head "offline" through all this - but you are right. Server side isn't offline.

will use.